README.md

```diff
@@ -84,3 +84,10 @@ Test the new image by starting a new user server! If you already had one running
 If you find new issues, you can revert back to the previous image by entering the old image name, back in the JupyterHub Configurator.
 
 *This will be streamlined in the future.*
+
+## Push image to a registry other than Quay.io
+
+The [jupyterhub/repo2docker-action](https://github.com/jupyterhub/repo2docker-action) can build and push the image to registries other than [Quay.io](https://quay.io/). Checkout the [action docs](https://github.com/jupyterhub/repo2docker-action/blob/master/README.md) for the instructions on how to setup your workflow to push to: [AWS Elastic Container Registry](https://github.com/jupyterhub/repo2docker-action#push-repo2docker-image-to-amazon-ecr), [Google Container Registry](https://github.com/jupyterhub/repo2docker-action#push-repo2docker-image-to-google-container-registry) (deprecated but popular), [Google Artifact Registry](https://github.com/jupyterhub/repo2docker-action#push-repo2docker-image-to-google-artifact-registry) (preferred), [Azure Container Registry](https://github.com/jupyterhub/repo2docker-action#push-repo2docker-image-to-azure-container-registry).
+
+**Note:**
+For cloud provider-specific registries, if we are running the cluster on our projects, please contact the 2i2c team to give you credentials for it.
```

Comment:
@GeorgianaElena, just a quick follow-up question... how we are going to provide these credentials? I guess this is also part of the recent conversations we had about secure means to get credentials from the communities...

Reply:
@damianavila I think that if we use something like `bitwarden` we could use their [send feature](https://bitwarden.com/products/send/) to securely send credentials with a link that would expire after some time.